modules/images-configuration-file.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration.adoc
+
+[id="images-configuration-file_{context}"]
+= Configuring image registry settings
```

Comment:
i don't love this title because people will probably assume it means the cluster registry when in reality there are fields here related to both external registries (CAs, allowed registries, etc)  as well as the cluster registry (internalRegistryHostname, externalRegistryHostname)